implementations/rust/ockam/ockam_channel/src/secure_channel_worker.rs

```diff
@@ -223,10 +223,10 @@ impl<V: SecureChannelVault, K: KeyExchanger + Send + 'static> SecureChannelWorke
         } else {
             return Err(SecureChannelError::InvalidInternalState.into());
         }
-        let _ = key_exchanger.process(payload.as_slice())?;
+        let _ = key_exchanger.handle_response(payload.as_slice())?;
```

Comment:
This also should be merged to develop not using this PR

Reply:
Also rebase